eth_crowdfund_be/eth_crowdfund_api/src/views/CampaignView.py

```diff
@@ -0,0 +1,74 @@
+from flask import request, json, Response, Blueprint, g, jsonify
+from ..models.Campaign import Campaign, CampaignSchema
+
+
+
+campaign_api = Blueprint('campaigns_api', __name__)
+campaign_schema = CampaignSchema()
+
+@campaign_api.route('/', methods=['POST'])
+def create():
+  req_data = request.get_json()
+  data = campaign_schema.load(req_data)
+  # data, error = campaign_schema.load(req_data)
```

Comment:
Ideally the `campaign_schema.load(req_data)` will represent the json payload of data, or an error. This however threw an error about too many arguments